src/beliefpropagation/beliefpropagation.jl

```diff
@@ -81,17 +81,25 @@ function belief_propagation_iteration(
   mts::DataGraph;
   contract_kwargs=(; alg="density_matrix", output_structure=path_graph_structure, maxdim=1),
   compute_norm=false,
+  update_order::String="parallel",
```

Comment:
I think it would be better to call it `update_sequence` to match our convention of using the term "contract sequence" in other parts of the code.